corehq/apps/accounting/tests/test_invoicing.py

```diff
@@ -145,34 +145,6 @@ def test_community_no_charges_no_invoice(self):
         self.assertRaises(Invoice.DoesNotExist,
                           lambda: Invoice.objects.get(subscription__subscriber__domain=domain.name))
 
-    def test_community_invoice(self):
```

Comment:
Might be good to take this opportunity to combine the test files `test_invoice_factory` and `test_invoicing` as it looks like there's an overlap of behavior and its difficult to determine what is responsible for testing what

Reply:
Will plan to work on that separately, but considering it out of scope for this PR.